packages/tools/src/utilities/contours/reverseIfAntiClockwise.ts

```diff
@@ -19,32 +9,11 @@ function getSumReducer(total: number, num: number): number {
  * @returns The contour, corrected to be clockwise if appropriate.
  */
 export default function reverseIfAntiClockwise(points: Types.Point2[]) {
-  const length = points.length;
-  if (!length) {
-    return points;
-  }
-  let xSum = 0;
-  for (const point of points) {
-    xSum += point[0];
-  }
-  const xMean = xSum / length;
-
-  let checkSum = 0;
+  const signedArea = getSignedArea(points);
```

Comment:
The previous function was a bit slower and it was also looping through all points twice. `checksum` was actually the double of the signed area.